### ISSUE TITLE
Standalone file?

### ISSUE THREAD

**Issue by jennybc** [r-lib/rematch2#19]
Have you thought about offering rematch2 as a standalone file? If so, here or from rlang? 🤔

**Comment by gaborcsardi** [comment]
If you want the main function, you can use this file: https://github.com/r-lib/pkgdepends/blob/main/R/rematch2.R

**Comment by jennybc** [comment]
Yeah I do already inline this in a few places. This issue is about whether to adopt our emerging conventions re: standalone files, in terms of file name, frontmatter, etc.

**Comment by gaborcsardi** [comment]
It is one short function, so that seems overkill to me but if you want to do it in a pr, I'll be happy to merge it.
